From 2cf6e8bec3e4f9ec5115276b78b8e1733fd7e39a Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 4 Dec 2024 18:02:25 +0000
Subject: [PATCH 680/697] arm64: dts: broadcom: Add dma-names for HDMI audio
 DMA

The frameworks seem to want it now, so make it happy.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712-rpi.dtsi | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi.dtsi
@@ -252,10 +252,12 @@ pciex4: &pcie2 { };
 
 &hdmi0 {
 	dmas = <&dma40 (10|(1<<30)|(1<<24)|(10<<16)|(15<<20))>;
+	dma-names = "audio-rx";
 };
 
 &hdmi1 {
 	dmas = <&dma40 (17|(1<<30)|(1<<24)|(10<<16)|(15<<20))>;
+	dma-names = "audio-rx";
 };
 
 &spi10 {
